include/pisa/util/progress.hpp

```diff
@@ -9,7 +9,7 @@ namespace pisa {
 
 class progress {
   public:
-    progress(const std::string& name, size_t goal) : m_name(name)
+    progress(const std::string& name, size_t goal) : m_name(name), m_elapsed()
     {
         if (goal == 0) {
             throw std::runtime_error("goal must be positive");
```

Comment:
As long as we're making changes to this file, can we also update this message to something more informative? Say, "Progress bar must have a positive goal but 0 given".